src/todo-input.ts

```diff
@@ -10,10 +10,10 @@ class TodoInputElement extends HTMLElement {
         this.update();
     }
 
-    handleInputChange(ev: Event) {
-        this.#value = ev.target.value;
+    handleInputChange(ev: KeyboardEvent) {
+        this.#value = this.text.value;
```

Comment:
This works fine but we are then ignoring the parameter received. Shoud we just remove it?